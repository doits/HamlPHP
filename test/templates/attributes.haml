!!! 5
%html
  %body
    %div(data-url = 'aasd')
      %p(class='paragraph' style='background: #fff') Hei
      %p(class='paragraph' style='background: #fff') Hei
      %p(class="paragraph" style="background: #fff; color: #000") Hei
      %p{'class' => 'paragraph'} Hei
      %p{this => 'att list',
        is => 'multiline'}
      %p{'id' => ['this', $item_here],
        has => $complex ? 'php' : "nothing",
        and => true,
        is_also => $multiline}