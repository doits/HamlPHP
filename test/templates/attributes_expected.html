<!DOCTYPE html>
<html>
  <body>
    <div data-url='aasd'>
      <p class="paragraph" style='background: #fff'>Hei</p>
      <p class="paragraph" style='background: #fff'>Hei</p>
      <p class="paragraph" style="background: #fff; color: #000">Hei</p>
      <p class="paragraph">Hei</p>
      <p this='att list' is='multiline'></p>
      <p <?php atts(array('id' => array('this', $item_here), 'has' => $complex ? 'php' : "nothing", 'and' => 'and', 'is_also' => $multiline)); ?>></p>
    </div>
  </body>
</html>
